build.sh

```diff
@@ -1,6 +1,4 @@
 mkdir -p build/
```

Comment:
The very first line of the bash script should start with `#!/bin/bash`. This instructs the operating system to use the bash interpreter.